spec/Dockerfile_spec.rb

```diff
@@ -137,7 +137,7 @@
         end
       end
 
-      describe docker_run_with_envs("consul_template_bootstrap_#{distro}", SERVICE_ENV: "peer-rise-runtime-1768", ALREADY_SET: "true") do
+      describe docker_run_with_envs("consul_template_bootstrap_#{distro}", PEER_ID: "peer-rise-runtime-1768", ALREADY_SET: "true") do
```

Comment:
You wan to leave `SERVICE_ENV` set as well, also `PEER_ID` is not `peer-rise-runtime-1768`, but rather `rise-runtime-1768`

Reply:
https://github.com/articulate/docker-consul-template-bootstrap/blob/master/entrypoint.sh#L17-L19